plugins/contact-resources/src/components/CreatePerson.svelte

```diff
@@ -74,35 +72,81 @@
   }
 
   let channels: AttachedData<Channel>[] = []
+
+  let matches: FindResult<Person> = []
+  $: findPerson(client, { ...object, name: combineName(firstName, lastName) }, channels).then((p) => {
+    matches = p
+  })
 </script>
 
 <Card
   label={contact.string.CreatePerson}
   okAction={createPerson}
-  canSave={firstName.length > 0 && lastName.length > 0}
+  canSave={firstName.length > 0 && lastName.length > 0 && matches.length === 0}
```

Comment:
May be we should use status for it and `<StatusControl slot="error" {status} />` like in createApplication?

Reply:
Nope, it show Person and allow to click.